stories/theme/core/Dropdown.css

```diff
@@ -36,5 +37,17 @@
 
 .Dropdown .Button-collapse:after {
   content: '\25bc';
+}
+
+.Dropdown .List {
+  padding: 5px;
+}
 
+.multiple .Text {
```

Comment:
If this is specific to a class outside of core, I'd much rather prefer it in the storybook theme.